site/content/in-dev/unreleased/policy.md

```diff
@@ -0,0 +1,207 @@
+  - Controls how long snapshots are retained before removal
+  - Applicable resources: Iceberg table, namespace, catalog 
+
+- **Custom policy types**: Can be defined for specific organizational needs (WIP)
+
+- **FGAC (Fine-Grained Access Control) policies**: Row filtering, column masking, column hiding (WIP)
+
+### Policy Inheritance
+
+The entity hierarchy in Polaris is structured as follows:
+
+```
+            Catalog
+               |
+           Namespace
+               |
+   +-----------+----------+
+   |           |          |
+Iceberg     Iceberg     Generic
+ Table       View        Table
+```
+
+Policies can be attached at any level, and inheritance flows from catalog down to namespace, then to tables and views.
+
+Policies can be inheritable or non-inheritable:
+
+- **Inheritable policies**: Apply to the target resource and all its applicable child resources
+- **Non-inheritable policies**: Apply only to the specific target resource
+
+The inheritance follows an override mechanism:
+1. Table-level policies override namespace and catalog policies
+2. Namespace-level policies override parent namespace and catalog policies
+
+## Working with Policies
+
+### Creating a Policy
+
+To create a policy, you need to provide a name, type, and optionally a description and content:
+
+```json
+POST /polaris/v1/{prefix}/namespaces/{namespace}/policies
+{
+  "name": "compaction-policy",
+  "type": "system.data-compaction",
+  "description": "Policy for optimizing table storage",
+  "content": "{\"version\": \"2025-02-03\", \"enable\": true, \"config\": {\"target_file_size_bytes\": 134217728}}"
+}
+```
+
+The policy content is validated against a schema specific to its type. Here are a few policy content examples
+- Data Compaction Policy
+```json
+{
+  "version": "2025-02-03",
+  "enable": true,
+  "config": {
+    "target_file_size_bytes": 134217728,
+    "compaction_strategy": "bin-pack",
+    "max-concurrent-file-group-rewrites": 5
+  }
+}
+```
+- Orphan File Removal Policy
+```json
+{
+  "version": "2025-02-03",
+  "enable": true,
+  "max_orphan_file_age_in_days": 30,
+  "locations": ["s3://my-bucket/my-table-location"],
+  "config": {
+    "prefix_mismatch_mode": "ignore"
+  }
+}
+```
+
+### Attaching Policies to Resources
+
+Policies can be attached to different resource levels:
+
+1. **Catalog level**: Applies to the entire catalog
+2. **Namespace level**: Applies to a specific namespace
+3. **Table-like level**: Applies to individual tables or views
+
+Example of attaching a policy to a table:
+
+```json
+PUT /polaris/v1/{prefix}/namespaces/{namespace}/policies/{policy-name}/mappings
+{
+  "target": {
+    "type": "table-like",
+    "path": ["NS1", "NS2", "test_table_1"]
+  }
+}
+```
+
+For inheritable policies, only one policy of a given type can be attached to a resource. For non-inheritable policies, multiple policies of the same type can be attached.
```

Comment:
[not requesting change]: is there a document about why this is the case?

Reply:
Added a note at the section "Policy Inheritance"